extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -1,9 +1,9 @@
 {
   "afsanalytics": {
-   "host": "//$server.afsanalytics.com",
-   "base": "//$server.afsanalytics.com/cgi_bin/",
-   "pageview": "//$server.afsanalytics.com/cgi_bin/connect.cgi?usr=$websiteidPauto&js=1&amp=1&title=_title_&url=_canonical_url_&refer=_document_referrer_&resolution=_screen_width_x_screen_height_&color=_screen_color_depth_&Tips=_random_",
-   "click":   "//$server.afsanalytics.com/cgi_bin/click.cgi?usr=$websiteid&event=$event&exit=$clicklabel"
+   "host": "//www.afsanalytics.com",
+   "base": "//www.afsanalytics.com/cgi_bin/",
+   "pageview": "//www.afsanalytics.com/cgi_bin/connect.cgi?usr=xxxxxxxxPauto&js=1&amp=1&title=_title_&url=_canonical_url_&refer=_document_referrer_&resolution=_screen_width_x_screen_height_&color=_screen_color_depth_&Tips=_random_",
```

Comment:
the `xxxx` comes from https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/vendors.js#L77. 

Similarly, all the changes in this file are just the test using the full config and being more smart about what requests will be sent.